### PR TITLE
Feature/project detail card api

### PR DESCRIPTION
**Pull request by jenna1k**

### CODE REVIEW

File: src/components/project/CircleImg.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+function CircleImg(props) {
+  return (
+    <img
+      className='card-img-top'
+      style={{
+        width: '3rem',
```
Comment (aliashkevich) [inline src/components/project/CircleImg.js]: Could you please place css in separate .css file?

File: src/components/project/ClientInfo.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  render() {
+    return (
+      <div className='col-lg-4 col-md-12 col-sm-12'>
+        <div className='card card-stats'>
+          <div className='card-header card-header-info card-header-icon'>
+            <div className='card-icon'>
+              <i className='material-icons'>location_city</i>
+            </div>
+            <div className='container-fluid' style={{padding: '10px'}}>
+              <p className='card-category' style={{paddingBottom: '5px'}}>
```
Comment (aliashkevich) [inline src/components/project/ClientInfo.js]: In this file also, please place css in separate file.

File: src/components/project/Participants.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    const users = this.state.users;
+
+    const participants = [];
+    if (this.state.loading == false) {
+      users.map(user => {
+        participantsArr.find((id, i) => {
+          if (id === user.id) return participants.push(user);
+        });
+      });
+      console.log('participants: ', participants);
```
Comment (aliashkevich) [inline src/components/project/Participants.js]: please, remove console.log

File: src/components/project/Participants.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    }
+    return (
+      <div className='col-lg-4 col-md-12 col-sm-12'>
+        <div className='card card-stats'>
+          <div className='card-header card-header-info card-header-icon'>
+            <div className='card-icon'>
+              <i className='material-icons'>people</i>
+            </div>
+            <div className='container-fluid' style={{padding: '10px'}}>
```
Comment (aliashkevich) [inline src/components/project/Participants.js]: Styles should be in separate .css file

File: src/components/project/Participants.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    }
+    return (
+      <div className='col-lg-4 col-md-12 col-sm-12'>
+        <div className='card card-stats'>
+          <div className='card-header card-header-info card-header-icon'>
+            <div className='card-icon'>
+              <i className='material-icons'>people</i>
+            </div>
+            <div className='container-fluid container-padding'>
+              <p className='card-category title-padding'>participants</p>
```
Comment (wilsoncooked) [inline src/components/project/Participants.js]: Participants not participants :)

File: src/components/project/Participants.js [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  }
+
+  render() {
+    const {participantsArr} = this.props;
+    const users = this.state.users;
+
+    const participants = [];
+    if (this.state.loading == false) {
+      users.map(user => {
+        participantsArr.find((id, i) => {
```
Comment (aliashkevich) [inline src/components/project/Participants.js]: Please, remove unused param.

File: src/components/project/Participants.js [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+  componentDidMount() {
+    this.getUsers();
+  }
+
+  render() {
+    const {participantsArr} = this.props;
+    const users = this.state.users;
+
+    const participants = [];
+    if (this.state.loading == false) {
```
Comment (wilsoncooked) [inline src/components/project/Participants.js]: I hear the === is better than == and more secure

**Review by wilsoncooked — COMMENTED** [review]
So cool how you're are passing props and it looks great!

**Review by wilsoncooked — APPROVED** [review]

**Review by aliashkevich — APPROVED** [review]

**Pull request merged**